### ISSUE TITLE
ERROR:  Event Gateway Response Code: 400

### ISSUE THREAD

**Issue by NorthernMan54** [NorthernMan54/homebridge-alexa#186]
Event gateway message for the Garage door is being sent when the garage is opening / closing and does not include an actual status.

`Feb 19 21:35:25 ip-172-26-3-93 startup[19162]: eventGW Error:  {"header":{"namespace":"System","name":"Exception","messageId":"ASRLS_cee1feca-a807-4064-836e-495fbf3adba4_1550612125342.0..ASRLS_cee1feca-a807-4064-836e-495fbf3adba4_1550612125342"},"payload":{"code":"INVALID_REQUEST_EXCEPTION","description":"Payload conversion failed."}} 400`

```
2019-02-19T21:35:25.166Z alexaActions Event {"host":"192.168.1.202","port":51827,"aid":4,"iid":16,"status":2}
2019-02-19T21:35:25.166Z alexaLocal Sending message {"context":{},"event":{"header":{"messageId":"05b92c15-feb6-4ec2-84a3-1d0cfb9a51f9","namespace":"Alexa","name":"ChangeReport","payloadVersion":"3"},"endpoint":{"endpointId":"QUM6MjI6M0Q6RTM6Q0U6MzEtTGVvbmFyZC1BbWF6b24tV1NTRU5TT1ItR2FyYWdlIERvb3ItMDAwMDAwNDEtMDAwMC0xMDAwLTgwMDAtMDAyNkJCNzY1Mjkx"},"payload":{"change":{"cause":{"type":"PHYSICAL_INTERACTION"},"properties":[{"namespace":"Alexa.ContactSensor","name":"detectionState","timeOfSample":"2019-02-19T21:35:25.166Z","uncertaintyInMilliseconds":500}]}}}}

```
Valid Message

```
2019-02-19T21:35:34.807Z alexaActions Event {"host":"192.168.1.202","port":51827,"aid":4,"iid":16,"status":0}
2019-02-19T21:35:34.808Z alexaLocal Sending message {"context":{},"event":{"header":{"messageId":"ac7c4f93-372e-4964-b953-f74e45574284","namespace":"Alexa","name":"ChangeReport","payloadVersion":"3"},"endpoint":{"endpointId":"QUM6MjI6M0Q6RTM6Q0U6MzEtTGVvbmFyZC1BbWF6b24tV1NTRU5TT1ItR2FyYWdlIERvb3ItMDAwMDAwNDEtMDAwMC0xMDAwLTgwMDAtMDAyNkJCNzY1Mjkx"},"payload":{"change":{"cause":{"type":"PHYSICAL_INTERACTION"},"properties":[{"namespace":"Alexa.ContactSensor","name":"detectionState","value":"DETECTED","timeOfSample":"2019-02-19T21:35:34.807Z","uncertaintyInMilliseconds":500}]}}}}
```

**Comment by sherazan** [comment]
Hi, I'm receiving again thi error:

[2019-4-5 00:33:35] [Alexa] ERROR:  Event Gateway Response Code: 400

I already disconnected and linked again the plugin, but the error is displayed again.
Do you have suggests?
Thanks

**Comment by NorthernMan54** [comment]
I was looking at the logs and I think it is due to the above and can be ignored.  Do you what device type it is from?

**Comment by sherazan** [comment]
Sorry, pheraps a little bit of confusion: I never sent a log about this error...

**Comment by NorthernMan54** [comment]
This is fixed with release "v0.4.13"
